ui/plugins/search/src/components/routes/search-page.tsx

```diff
@@ -162,12 +182,40 @@ const SearchPage: React.FC<SearchPageProps> = props => {
     history.back();
   };
 
-  const searchCount = searchState
-    ? searchState.profiles?.length +
-      searchState.entries?.length +
-      searchState.tags?.length +
-      searchState.comments?.length
-    : 0;
+  const searchCount =
+    searchProfilesState?.length ||
```

Comment:
I think this does the same thing? Why `0 +` is needed?
```
searchProfilesState?.length ||
    searchPostsState?.length ||
    searchCommentsState?.length ||
    searchTagsState?.length ||
    0;
    ```

Reply:
Because you need to add the number of results from each of the search topics, but each of them could be undefined at various stages

Reply:
got confused by the formatting, lol

Reply:
prettier made a bit of a mess of it

Reply:
you can use also nullish coalescing operator like 

```
searchProfilesState?.length ?? 0
+ searchPostsState?.length ?? 0
+ searchCommentsState?.length ?? 0
....